### ISSUE TITLE
Autoarchive messing up global database

### ISSUE THREAD

**Issue by Cronycs** [Brummolix/AutoarchiveReloaded#26]
Hello,
I am reporting an issue I have with Autoarchive. This is an issue I have since many versions, but I am currently using Thunderbird 60.0 and AutoarchiveReloaded 0.9.9.6.
Each time after a manual auto archive, all emails that have been archive are missing in the global database and therefore can't be found with the global search of Thunderbird.
I have to follow this procedure to restore a healthy global database : https://support.mozilla.org/en-US/kb/rebuilding-global-database
I hope the feedback can be useful.
Thank you for this great addon !

**Comment by Brummolix** [comment]
I guess this is a bug of Thunderbird. It can not be controlled by AutoarchiveReloaded.
I guess it also happens if you mark multiple messages and archive them manually (for example with Thunderbird context menu)? If so, please report the bug to Thunderbird itself.

**Comment by wsmwk** [comment]
@Cronycs have you reported this in bugzilla?

**Comment by wsmwk** [comment]
Yes, this is a bug in Thunderbird https://bugzilla.mozilla.org/show_bug.cgi?id=1205248
